packages/app-root/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@zooniverse/app-root",
+  "description": "",
+  "license": "Apache-2.0",
+  "author": "Zooniverse <contact@zooniverse.org> (https://www.zooniverse.org/)",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "type": "module",
+  "dependencies": {
+    "@zooniverse/async-states": "~0.0.1",
+    "@zooniverse/grommet-theme": "~3.1.1",
+    "@zooniverse/panoptes-js": "~0.4.1",
+    "@zooniverse/react-components": "~1.6.1",
+    "grommet": "~2.33.2",
+    "grommet-icons": "~4.11.0",
+    "next": "13.5.2",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
```

Comment:
Why are these pinned to specific versions?

Reply:
I'll change react versions to a ~, but pin Next.js to 13.5.3 as discussed in our call this morning. Next.js 13.5.4 breaks storybook, and we want app-content-pages and app-root to be on the same version of Next.js